services/src/main/scala/cromwell/services/metadata/WorkflowQueryKey.scala

```diff
@@ -180,6 +164,14 @@ object WorkflowQueryKey {
     override def displayName = "include subworkflows"
     override def defaultBooleanValue: Boolean = true
   }
+
+  // Note: `false` means "oldest first" not "no ordering specified". Might want to encode in code later.
```

Comment:
➕

Reply:
Will address later. Want to merge this before I accumulate any more conflicts.